### ISSUE TITLE
Increase user confidence in engaging with the leaderboard

### ISSUE THREAD

**Issue by megan-godfrey** [openflighthpc/carbon-leaderboard#32]
Feedback from ISC suggests users weren't confident talking about carbon emissions. If a user is unsure about the carbon emissions of their system, this may have made them less likely to want to submit their data onto a public leaderboard. ('What if my servers have the worst emissions and I go to the bottom of the leaderboard?')

Therefore, to increase engagement with the leaderboard, we could add information and tools for helping a user gain confidence in their knowledge of the carbon emissions of their system.

Ideas for doing this:

- Allow submitting server data without automatically entering it onto the leaderboard. A user could investigate the carbon emissions of their device, and compare these to devices entered onto the leaderboard, without this data becoming publicly available.
- Add a page with information about the emissions associated with HPC systems. This would allow users to assess, theoretically, how high or low their emissions might be. This could include information from the 'Greener HPC' page in the openflight docs.
- Add data from a few servers that have high emissions, so new users' entries are unlikely to go to the bottom of the leaderboard.

**Comment by Womblue** [comment]
Some of these issues might also be mitigated when we start to include different leaderboard ranking metrics - right now we're ranking per core at max load, but a server near the bottom of the leaderboard by those rankings could be quite efficient at half load or minimum load (for example)

**Comment by megan-godfrey** [comment]
Yeah true :) I think that would help in conjunction with these ideas, and might make users more likely to submit their data once they know what their rankings would be. E.g. a user has no idea how good their emissions are -> submits data -> sees that they're doing alright at least on a couple of leaderboards -> feels confident enough to enter their data onto the public leaderboard.
